src/proposals/20250320/SparkEthereum_20250320.sol

```diff
@@ -0,0 +1,271 @@
+        });
+
+        listings[3] = IEngine.Listing({
+            asset:       Ethereum.RSETH,
+            assetSymbol: 'rsETH',
+            priceFeed:   RSETH_ORACLE,
+            rateStrategyParams:                Rates.RateStrategyParams({
+                optimalUsageRatio:             _bpsToRay(45_00),
+                baseVariableBorrowRate:        _bpsToRay(5_00),
+                variableRateSlope1:            _bpsToRay(15_00),
+                variableRateSlope2:            _bpsToRay(300_00),
+                stableRateSlope1:              0,
+                stableRateSlope2:              0,
+                baseStableRateOffset:          0,
+                stableRateExcessOffset:        0,
+                optimalStableToTotalDebtRatio: 0
+            }),
+            enabledToBorrow:       EngineFlags.DISABLED,
+            stableRateModeEnabled: EngineFlags.DISABLED,
+            borrowableInIsolation: EngineFlags.DISABLED,
+            withSiloedBorrowing:   EngineFlags.DISABLED,
+            flashloanable:         EngineFlags.DISABLED,
+            ltv:                   72_00,
+            liqThreshold:          73_00,
+            liqBonus:              10_00,
+            reserveFactor:         15_00,
+            supplyCap:             2_000,
+            borrowCap:             0,
+            debtCeiling:           0,
+            liqProtocolFee:        10_00,
+            eModeCategory:         0
+        });
+
+        return listings;
+    }
+
+    function collateralsUpdates()
+        public pure override returns (IEngine.CollateralUpdate[] memory collateralUpdates)
+    {
+        collateralUpdates = new IEngine.CollateralUpdate[](1);
+
+        collateralUpdates[0] = IEngine.CollateralUpdate({
+            asset:          Ethereum.CBBTC,
+            ltv:            EngineFlags.KEEP_CURRENT,
+            liqThreshold:   EngineFlags.KEEP_CURRENT,
+            liqBonus:       EngineFlags.KEEP_CURRENT,
+            debtCeiling:    EngineFlags.KEEP_CURRENT,
+            liqProtocolFee: EngineFlags.KEEP_CURRENT,
+            eModeCategory:  3
+        });
+    }
+
+    function _postExecute() internal override {
+        // Seed the new LBTC pool
+        IERC20(Ethereum.LBTC).approve(address(LISTING_ENGINE.POOL()), 0.0001e8);
+        LISTING_ENGINE.POOL().supply(Ethereum.LBTC, 0.0001e8, address(this), 0);
+
+        // Seed the new tBTC pool
+        IERC20(Ethereum.TBTC).approve(address(LISTING_ENGINE.POOL()), 0.0001e18);
+        LISTING_ENGINE.POOL().supply(Ethereum.TBTC, 0.0001e18, address(this), 0);
+
+        // Seed the new ezETH pool
+        IERC20(Ethereum.EZETH).approve(address(LISTING_ENGINE.POOL()), 0.0001e18);
+        LISTING_ENGINE.POOL().supply(Ethereum.EZETH, 0.0001e18, address(this), 0);
+
+        // Seed the new rsETH pool
+        IERC20(Ethereum.RSETH).approve(address(LISTING_ENGINE.POOL()), 0.0001e18);
+        LISTING_ENGINE.POOL().supply(Ethereum.RSETH, 0.0001e18, address(this), 0);
```

Comment:
These are not high enough, will be ~$0.2. This actually revealed an issue in our test:

https://github.com/sparkdotfi/spark-spells/blob/d350e4236a6a2dd34fed853811c5eaa18eeff780/src/test-harness/SparklendTests.sol#L196

This line should be 

```solidity
uint256 poolValue = aToken.totalSupply() * priceOracle.getAssetPrice(reserves[i]) * 1e6 / (10 ** aToken.decimals()) / 1e8;
```

Reply:
Looking at this test again, it doesn't really make sense to be anchoring to a USD price. This test here is to prevent share inflation attacks, but we ran into an issue where it was set too high for BTC. I think what we can do is convert it back to the previous version and just lower the precision in absolute terms.

1e14 is definitely enough precision to prevent share inflation it doesn't matter the price is below $1.

Reply:
I've switched back to the old version and lowered to 1e4 requirement which is still sufficient to prevent share inflation attack.